packages/config-loader/src/loader.ts

```diff
@@ -26,39 +33,22 @@ export type LoadConfigOptions = {
   // Absolute paths to load config files from. Configs from earlier paths have lower priority.
   configPaths: string[];
 
-  // TODO(Rugvip): This will be removed in the future, but for now we use it to warn about possible mistakes.
-  env: string;
-};
-
-class Context {
-  constructor(
-    private readonly options: {
-      env: { [name in string]?: string };
-      rootPath: string;
-    },
-  ) {}
-
-  get env() {
-    return this.options.env;
-  }
+  /** @deprecated This option has been removed */
```

Comment:
Being deprecated, it hasn't technically been removed yet, is that right? should this @tag also point the user say to the doco that's been updated via this PR to guide devs to what to replace it with? (e.g., the new section on the `${VAR}` including from env)

Reply:
This is for the old APP_ENV-based config selection, which has been deprecated for a long time and was mostly removed in this PR. This option wasn't properly deprecated back then though, which was an oversight, and so we can't remove it now, just deprecate it. All docs etc are long gone tho (I think 😅)